tests/test_diffraction_objects.py

```diff
@@ -233,7 +233,10 @@ def test_init_invalid_xtype():
             },
             {"xtype": "tth", "yarray": np.array([1, 2, 2.5, 3, 6, 10])},
         ),
-        (  # C3: Same length x-arrays with approximate x-value match
+        (  # C3: Different x-arrays with same length,
+            # x-value has closest match at q=0.12 (y=10) and q=0.14 (y=1)
+            # for original and target diffraction objects,
+            # expect y-array to divide by 10
```

Comment:
ok - yarray of 10 and 1 are compared since they are the nearest values at q=0.1. Since the second is 1/10 of the original, we expect y value to be 1/10